crates/router/src/core/routing.rs

```diff
@@ -818,6 +817,37 @@ pub async fn retrieve_linked_routing_config(
     }
 }
 
+pub async fn create_connector_agnostic_mandate_config(
```

Comment:
Make this an `upsert` logic instead.
```rust
let config = state.find_config();
match config {
    Ok(conf) => { update },
    Err(err) if err.is_db_not_found() => { insert },
    Err(err) => { throw error }
}
```

Reply:
additionally, add logic to the "create business profile" core function to insert a default value to the configs table from the start.

Reply:
This function `find_config_by_key_unwrap_or` has the same logic.